mux.go

```diff
@@ -58,10 +58,18 @@ type BaseRoute struct {
 	Middlewares          []func(http.Handler) http.Handler
 	AcceptedContentTypes []string // Content types accepted for the request body. If nil, all content types (*/*) are accepted.
 	Hidden               bool     // If true, the route will not be documented in the OpenAPI spec
+	OverrideDescription  bool     // Override the default description
```

Comment:
Only thing I can say is. Does this really need to be exported?

Reply:
I agree